src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -528,4 +528,34 @@ public function findCartByTokenValueAndChannel(string $tokenValue, ChannelInterf
             ->getOneOrNullResult()
         ;
     }
+
+    public function sumPaidOrdersByChannelInPeriodGroupedByYearAndMonth(
```

Comment:
https://github.com/Sylius/Sylius/blob/bb26e46f3c1d7af54815ecdbb3f9c8003c2a5ae6/src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php#L308-L326

Since methods are almost identical, they should have the same name scheme

```suggestion
    public function getTotalPaidSalesForChannelInPeriodGroupedByYearAndMonth(
```